modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -32,29 +32,37 @@
   (with-open [_ (prom/timer generate-duration-seconds "json")]
     (fhir-spec/unform-json body)))
 
-(defn- generate-xml** [body]
+(defn- xml-byte-array [body]
   (let [out (ByteArrayOutputStream.)]
     (with-open [writer (io/writer out)]
       (xml/emit (fhir-spec/unform-xml body) writer))
     (.toByteArray out)))
 
-(defn- generate-xml* [body]
+(defn- e->xml-byte-array [e]
```

Comment:
if the body if this function doesn't fit into the line it is called, than please name if `generate-xml-error`

Reply:
But it's not generating any error. This function is simply a pipeline that *converts* an exception into an xml-byte-array (or however we end up calling the other function's result).

Reply:
I can certainly rename it to `e->whatever`, but it's a pipeline function. That's why I used [the `->` convention](https://guide.clojure.style/#naming-conversion-functions).
Or would you suggest doing this differently?

Reply:
The -> convention is about a conversion function, not about a pipeline function. Here you can say that an error is converted into a byte array containing the error in XML format. But I like `generate-xml` better than just saying that we convert some data type into another data type. Generating is like the inverse of parsing. 

And please don't try to change everything, if the task is just to add a 500 status code.